CodeStarterCamp_Week3/main.swift

```diff
@@ -17,10 +17,21 @@ enum Coffee {
 }
 
 class Person {
-    var money: Int = 0
+    var money: Int
     
-    func order(_ coffee: Coffee, coffeeShop: CoffeeShop) {
-        coffeeShop.make(coffee, barista: misterLee)
+    init(money: Int) {
+        self.money = money
+    }
+    
+    func order(_ coffee: Coffee, coffeeShop: CoffeeShop, name: String) {
+        guard let price = coffeeShop.menu[coffee] else { return print("상품이 존재하지 않습니다.") }
```

Comment:
일반적으로는 다음과 같이 작성하는 것 같습니다:
```suggestion
        guard let price = coffeeShop.menu[coffee] else {
            print("상품이 존재하지 않습니다.")
            return
        }
```
* print문보다 return이 아래에 위치할 수 있게
* 두 단어 이상일 때는 개행을 하게

이렇게 작성하는게 일반적으로 보입니다.